vm-builder/files/vminit

```diff
@@ -25,6 +25,9 @@ mount -t proc  proc  /proc
 mount -t sysfs sysfs /sys
 mount -t cgroup2 cgroup2 /sys/fs/cgroup
 
+# change kernel log level to DEBUG
+echo 7 > /proc/sys/kernel/printk
```

Comment:
Can we do this earlier on? Maybe nitpicking, but I worry about missing logs due to earlier changes (e.g., udev rules) taking effect before this change

Reply:
No we can't, procfs is mounted right before :)

Reply:
Ah! Fair enough, I guess :sweat_smile: